qiita_db/support_files/patches/19.sql

```diff
@@ -0,0 +1,34 @@
+-- March 19, 2015
+-- Removes the status from the study and adds it to the processed data table.
+
+-- Modify the study_status_table so it becomes the processed_data_status table
+ALTER TABLE qiita.study_status RENAME TO processed_data_status;
+ALTER TABLE qiita.processed_data_status RENAME COLUMN study_status_id TO processed_data_status_id;
+ALTER TABLE qiita.processed_data_status RENAME COLUMN status TO processed_data_status;
```

Comment:
Since this is an open discussion and we have to agree in what are we going to standardize, I'm not going to change this and if it has to be fixed, it will be done as part of fixing the issue #292